polishNotation.py

```diff
@@ -0,0 +1,39 @@
+def polishNotation(expression):
+    s = stack()
+    result = 0
+    for exp in range(0, len(expression)-1):
+        exp_list.append(exp)
+
+    reverse_exp_list = exp_list[::-1]

```

Comment:
This uses O(N) memory for no good reason. You can always iterate over a list backwards:

for i in range(len(expression) - 1, -1, -1):

You should be super confident with range as this is a paramount function. Read, re-read and toy with all uses of range: https://docs.python.org/3/library/stdtypes.html#typesseq-range